### DISCUSSION TITLE
Nested App with Turborepo and Next.JS

### DISCUSSION THREAD

**Discussion by ivanhueso** [vercel/turborepo#2996] [Help] [answered]
I would like to know if there is a way to nest an app inside another one. 

For Instance: 
  I Have a **Turborepo** with two apps. Shell App and  Auth App (with it's own apis). 
  I would like to know how can i nest Auth App inside `shellApp/auth`

Thank you.

**Comment by anthonyshew** [comment, marked as answer]
This is a domain-driven packaging pattern. Instead of designing a package around `ui`, you would design it around `shell` or similar. You would then import the implementations into your app as you need.
